src/nvim/search.c

```diff
@@ -4289,7 +4289,7 @@ static void search_stat(int dirc, pos_T *pos,
 
       if (curwin->w_p_rl && *curwin->w_p_rlc == 's') {
         if (cur == OUT_OF_TIME) {
-          vim_snprintf(t, SEARCH_STAT_BUF_LEN, "[?/??]");
+          vim_snprintf(t, SEARCH_STAT_BUF_LEN, "[??/?]");
```

Comment:
When the compiler sees two consecutive `?`, it considers that the start of a trigraph.  To avoid that, simply escape one of them.

```suggestion
          vim_snprintf(t, SEARCH_STAT_BUF_LEN, "[\??/?]");
```

Reply:
Learned something new, thanks!

Reply:
https://en.wikipedia.org/wiki/Digraphs_and_trigraphs#C so, according to wikipedia, the second `?` should be escaped